flink-streaming-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducerBase.java

```diff
@@ -374,4 +380,11 @@ public static Properties getPropertiesFromBrokerList(String brokerList) {
 		props.setProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList);
 		return props;
 	}
+
+	// this is exposed for testing purposes
+	protected long numPendingRecords() {
```

Comment:
I don't think its available in `release-1.1`, is it? I might be mistakened, though ..